### PR TITLE
documentation: Update README and add CoC & CONTRIBUTING

### PR DESCRIPTION
**Pull request by EdDev**
Content has been inspired by the following projects:
- https://github.com/containernetworking/plugins
- https://github.com/kubevirt/kubevirt

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+Contributor Covenant Code of Conduct
+====================================
```
Comment (phoracek) [inline CODE_OF_CONDUCT.md]: Please unify the formatting you use for headings

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+of experience, nationality, personal appearance, race, religion, or sexual
+identity and orientation.
+
+Our Standards
+-------------
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
```
Comment (phoracek) [inline CODE_OF_CONDUCT.md]: Please unify the formatting you use for bullet points

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+and you might get asked to resubmit the PR or divide the changes into more than
+one PR/commit.
+
+### Format of the Commit Message
+
+We follow a rough convention for commit messages that is designed to answer two
+questions: what changed and why. The subject line should feature the what and
+the body of the commit should describe the why.
+
+See https://chris.beams.io/posts/git-commit/
```
Comment (phoracek) [inline CONTRIBUTING.md]:
```suggestion
See https://chris.beams.io/posts/git-commit/.
```

**Review by phoracek — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (EdDev) [reply CODE_OF_CONDUCT.md]: Done

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (EdDev) [reply CODE_OF_CONDUCT.md]: Done

File: CONTRIBUTING.md (hunk 3 above)
Comment (EdDev) [reply CONTRIBUTING.md]: Done

**Review by qinqon — CHANGES_REQUESTED** [review]
Add a reference to the godoc https://pkg.go.dev/github.com/eddev/go-nft

**Review by qinqon — APPROVED** [review]

**Pull request merged**